modules/backend/widgets/Lists.php

```diff
@@ -1698,6 +1710,84 @@ public function onToggleTreeNode()
         return $this->onRefresh();
     }
 
+    //
+    // Internal
+    //
+
+    /**
+     * Looks at the model for defined options.
+     *
+     * @param $field
+     * @param $fieldOptions
+     * @return mixed
+     */
+    protected function getOptionsFromModel($field, $fieldOptions, $fieldName)
+    {
+        /*
+         * Advanced usage, supplied options are callable
+         */
+        if (is_array($fieldOptions) && is_callable($fieldOptions)) {
+            $fieldOptions = call_user_func($fieldOptions, $this, $field);
+        }
+
+        /*
+         * Refer to the model method or any of its behaviors
+         */
+        if (!is_array($fieldOptions) && !$fieldOptions) {
+
+            $methodName = 'get'.studly_case($fieldName).'Options';
```

Comment:
@SaifurRahmanMohsin Please add spaces between concatenation portions.
```suggestion
            $methodName = 'get' . studly_case($fieldName) . 'Options';
```